docs/blog/2022-11-16-corso-announcement-free-backup-for-microsoft-365.md

```diff
@@ -76,7 +76,7 @@ costs, as well.
 ## Interested in Trying Corso?
 
 Corso, currently in alpha, provides a CLI-based tool for backups of your M365 data.
-[Follow the quickstart guide](../docs/quickstart) to start protecting your business-critical M365 data in
+[Follow the quickstart guide](../../docs/quickstart/) to start protecting your business-critical M365 data in
```

Comment:
the trailing slash pattern here is different from what I see below. Likely one of the two is off.

Reply:
Ah, somewhat of an accidental change that is actually a no-op.